core/cmd/shell_local.go

```diff
@@ -280,7 +280,7 @@ func (s *Shell) runNode(c *cli.Context) error {
 		lggr.Warn("Chainlink is running in DEVELOPMENT mode. This is a security risk if enabled in production.")
 	}
 
-	if err := utils.EnsureDirAndMaxPerms(s.Config.RootDir(), os.FileMode(0700)); err != nil {
+	if err = utils.EnsureDirAndMaxPerms(s.Config.RootDir(), os.FileMode(0700)); err != nil {
```

Comment:
Were the changes in this file intentional? This introduces a very small change in behaviour since we're now no longer shadowing the err variable and are reassigning it.

Reply:
I was debugging some lint issues and committed the changes 🤦